Manager/BackupManager.php

```diff
@@ -62,13 +63,22 @@ public function execute()
             // Transfer with all clients
             $this->cm->upload($this->processor->getArchivePath());
 
-            $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
         } catch (\Exception $e) {
-            // write log
+            // Write log
             $this->logger->critical('[dizda-backup] Unexpected exception.', array('exception' => $e));
 
             return false;
+        } finally {
```

Comment:
It was introduced in PHP5.5. http://php.net/manual/en/language.exceptions.php

Do you want me to rewrite it?